permission_handler/android/src/main/java/com/baseflow/permissionhandler/PermissionConstants.java

```diff
@@ -31,62 +31,41 @@
-            PERMISSION_GROUP_CALENDAR,
-            PERMISSION_GROUP_CAMERA,
-            PERMISSION_GROUP_CONTACTS,
-            PERMISSION_GROUP_LOCATION,
-            PERMISSION_GROUP_LOCATION_ALWAYS,
-            PERMISSION_GROUP_LOCATION_WHEN_IN_USE,
-            PERMISSION_GROUP_MEDIA_LIBRARY,
-            PERMISSION_GROUP_MICROPHONE,
-            PERMISSION_GROUP_PHONE,
-            PERMISSION_GROUP_PHOTOS,
-            PERMISSION_GROUP_REMINDERS,
-            PERMISSION_GROUP_SENSORS,
-            PERMISSION_GROUP_SMS,
-            PERMISSION_GROUP_SPEECH,
-            PERMISSION_GROUP_STORAGE,
-            PERMISSION_GROUP_IGNORE_BATTERY_OPTIMIZATIONS,
-            PERMISSION_GROUP_NOTIFICATION,
-            PERMISSION_GROUP_ACCESS_MEDIA_LOCATION,
-            PERMISSION_GROUP_ACTIVITY_RECOGNITION,
-            PERMISSION_GROUP_UNKNOWN,
-    })
+    @IntDef({PERMISSION_GROUP_CALENDAR, PERMISSION_GROUP_CAMERA, PERMISSION_GROUP_CONTACTS,
+            PERMISSION_GROUP_LOCATION, PERMISSION_GROUP_LOCATION_ALWAYS,
+            PERMISSION_GROUP_LOCATION_WHEN_IN_USE, PERMISSION_GROUP_MEDIA_LIBRARY,
+            PERMISSION_GROUP_MICROPHONE, PERMISSION_GROUP_PHONE, PERMISSION_GROUP_PHOTOS,
+            PERMISSION_GROUP_REMINDERS, PERMISSION_GROUP_SENSORS, PERMISSION_GROUP_SMS,
+            PERMISSION_GROUP_SPEECH, PERMISSION_GROUP_STORAGE,
+            PERMISSION_GROUP_IGNORE_BATTERY_OPTIMIZATIONS, PERMISSION_GROUP_NOTIFICATION,
+            PERMISSION_GROUP_ACCESS_MEDIA_LOCATION, PERMISSION_GROUP_ACTIVITY_RECOGNITION,
+            PERMISSION_GROUP_UNKNOWN, PERMISSION_GROUP_SYSTEM_ALERT_WINDOW,})
     @interface PermissionGroup {
     }
 
-    //PERMISSION_STATUS
+    // PERMISSION_STATUS
     static final int PERMISSION_STATUS_DENIED = 0;
     static final int PERMISSION_STATUS_GRANTED = 1;
     static final int PERMISSION_STATUS_RESTRICTED = 2;
     static final int PERMISSION_STATUS_NOT_DETERMINED = 3;
     static final int PERMISSION_STATUS_NEWER_ASK_AGAIN = 4;
 
     @Retention(RetentionPolicy.SOURCE)
-    @IntDef({
-            PERMISSION_STATUS_DENIED,
-            PERMISSION_STATUS_GRANTED,
-            PERMISSION_STATUS_RESTRICTED,
-            PERMISSION_STATUS_NOT_DETERMINED,
-            PERMISSION_STATUS_NEWER_ASK_AGAIN,
-    })
+    @IntDef({PERMISSION_STATUS_DENIED, PERMISSION_STATUS_GRANTED, PERMISSION_STATUS_RESTRICTED,
+            PERMISSION_STATUS_NOT_DETERMINED, PERMISSION_STATUS_NEWER_ASK_AGAIN,})
     @interface PermissionStatus {
     }
 
-    //SERVICE_STATUS
+    // SERVICE_STATUS
     static final int SERVICE_STATUS_DISABLED = 0;
     static final int SERVICE_STATUS_ENABLED = 1;
     static final int SERVICE_STATUS_NOT_APPLICABLE = 2;
 
     @Retention(RetentionPolicy.SOURCE)
-    @IntDef({
-            SERVICE_STATUS_DISABLED,
-            SERVICE_STATUS_ENABLED,
-            SERVICE_STATUS_NOT_APPLICABLE
-    })
+    @IntDef({SERVICE_STATUS_DISABLED, SERVICE_STATUS_ENABLED, SERVICE_STATUS_NOT_APPLICABLE})
```

Comment:
Please make sure all service status definitions are on their own line (like the previous version).